### ISSUE TITLE
Run `package-kubeapps-version.sh` automatically on bitnami chart release

### ISSUE THREAD

**Issue by absoludity** [vmware-tanzu/package-for-kubeapps#14]
The last step in the automation will be to run the carvel release automatically when the bitnami chart is released.

This is more of an infrastructure task, as it may need to run as part of the bitnami pipeline within the VPN.

**Comment by castelblanque** [comment]
It would be nice having every step automated.
Are we sure that we want to mirror the releases to Bitnami chart?
If package is not to be included until the next TCE version, it might not be that necessary.

**Comment by absoludity** [comment]
Yeah, it wouldn't need to be every bitnami chart release, perhaps just our Kubeapps releases, but note that running the script doesn't add the carvel package to TCE, it just generates, tests and publishes to the registry. We can still choose which versions get added to TCE.

**Comment by ppbaena** [comment]
> Yeah, it wouldn't need to be every bitnami chart release, perhaps just our Kubeapps releases, but note that running the script doesn't add the carvel package to TCE, it just generates, tests and publishes to the registry. We can still choose which versions get added to TCE.

+1. So we have it automated and tested to decide which version get added to TCE.
